build.fsx

```diff
@@ -1,16 +1,21 @@
 // --------------------------------------------------------------------------------------
-// FAKE build script 
```

Comment:
These files are largely mechanical. The next step here would be to move from the monolithic FAKE dependency to a paket build group with just the few modules we use (which mostly map 1-to-1 with the `open Fake.xxx` statements) and move to the Fake 5 netcore runner.